logfire/_internal/exporters/console.py

```diff
@@ -251,6 +251,18 @@ def _details_parts(self, span: Record, indent_str: str) -> TextParts:
         else:
             return []
 
+    def is_schema_valid(self, schema: dict[str, Any], value: Any) -> bool:
+        """Check if the schema is valid for the value."""
+        schema_type = schema.get('type') if schema else None
+        if schema_type:
+            if value is None:
+                return False
```

Comment:
```suggestion
```

you shouldn't need this now.

please test https://github.com/pydantic/logfire/pull/1289#issuecomment-3150200122